guide/inheritance-wallet/succession.md

```diff
@@ -0,0 +1,246 @@
+images_inheritance-transaction-create:
+    - file: inheritance-transaction-create/move-funds-app-home-initial
+      alt: App home screen showing the inheritance wallet.
+      caption: Christina goes into the inheritance wallet to create a transaction.
+    - file: inheritance-transaction-create/move-funds-wallet-overview
+      alt: Wallet overview screen.
+      caption: On the wallet details page, she hits "pay".
+    - file: inheritance-transaction-create/move-funds-recipient-filled
+      alt: Screen letting the user enter a payment address.
+      caption: She pastes the address of her personal bitcoin wallet.
+    - file: inheritance-transaction-create/move-funds-amount
+      alt: Screen letting the user choose the amount that should be sent.
+      caption: Christina wants to send 50% of the wallet balance.
+    - file: inheritance-transaction-create/move-funds-fees
+      alt: Screen letting the user choose the transaction fee.
+      caption: Christina goes with the default transaction speed. 
+    - file: inheritance-transaction-create/move-funds-review
+      alt: Review screen that shows a summary of the all transaction information.
+      caption: Christina reviews the details and then creates the transaction.
+    - file: inheritance-transaction-create/move-funds-transaction-created
+      alt: Screen showing a success dialog that lets the user choose whether they want to sign the transaction now or later.
+      caption: Christina goes on to sign the transaction. 
+images_inheritance-transaction-sign-christina:
+    - file: inheritance-transaction-sign/transaction-details
+      alt: Screen showing the transaction details.
+      caption: Christina checks the details again and scrolls down to find her signing device.
+    - file: inheritance-transaction-sign/move-funds-sign-christina-initial
+      alt: Screen showing the transaction details.
+      caption: Christina scrolls down to her inheritance key and hits "sign".
+    - file: inheritance-transaction-sign/move-funds-sign-christina-flow
+      alt: Placeholder screen representing the transaction signing flow.
+      caption: The app takes her through the signing flow.
+    - file: inheritance-transaction-sign/move-funds-sign-christina-done
+      alt: Screen showing the transaction details. It shows that one of the inheritance keys has already signed the transaction.
+      caption: The first signature has been provided.
+    - file: inheritance-transaction-sign/move-funds-sign-christina-share
+      alt: Screen showing a QR code and sharing options.
+      caption: Christina shares the partially signed transaction with David.
+images_inheritance-transaction-sign-david:
+    - file: inheritance-transaction-sign/move-funds-sign-david-payment-sheet
+      alt: Dialog showing three different payment options.
+      caption: David initiates the transaction signing process.
```

Comment:
In here, where did David have to click to get the dialog showing the 3 different options to create, sign, broadcast a txn?

I couldn't catch it. I'm assuming it was probably the QR button between the **Request** and **Pay** button shown at line 39 where Christina clicks on "pay" !!

Reply:
He tapped "pay", which opens the sheet. But you're right, in the mockups before, where Christina initiates the transaction, this step is missing. 😢